app/src/main/AndroidManifest.xml

```diff
@@ -714,6 +714,7 @@
             android:foregroundServiceType="connectedDevice|location"
             />
         <service android:name=".cgm.sharefollow.ShareFollowService" />
+        <service android:name=".cgm.connectfollow.ConnectFollowService" />
```

Comment:
Likely, you need the attribute `dataSync`([FOREGROUND_SERVICE_TYPE_DATA_SYNC](https://developer.android.com/reference/android/content/pm/ServiceInfo#FOREGROUND_SERVICE_TYPE_DATA_SYNC)) for Android 10+ and implement it at the service layer.

Reply:
Are you sure, that dataSync is really needed? I have used the existing Dexcom Share Follower and  Nightscout Follower as references during the implementation. I don't see any of them having the dataSync attribute. Currently I am using NSFollower on my Samsung S9+ with Android 10 and it is working perfectly.

Reply:
Are you sure that dataSync is required? I have used Dexcom Share follower and Nightscout follower as templates during the implementation and I don't see this in any of them. I am currently using xDrip+ NSFollower on a Samsung S9+ with Android 10 and I have no issues with it.

Reply:
Most of the services are implemented as foreground services and currently requesting `connectedDevice` and/or `location` in the code. That does not make sense for follower services. With Android 10+, only services requesting to run in the foreground can run in the foreground if the `foregroundServiceType` is set correctly in `AndroidManifest.xml`. Otherwise, they run but not always as a foreground service. See #1568 for a similar problem with connected devices.

Indeed, we have to fix this in general. Hence, keep it as it as for now.